packages/server/src/automations/tests/utilities/AutomationTestBuilder.ts

```diff
@@ -294,9 +387,8 @@ class AutomationBuilder extends BaseStepBuilder {
         stepNames: {},
       },
       type: "automation",
-      appId: options.appId ?? setup.getConfig().getAppId(),
+      appId: options.appId ?? this.config.getAppId(),
```

Comment:
This was incorrect, if you passed a `config` but not an `appId` it would try to pull an `appId` from the uninitialised `setup.getConfig()`, resulting in an error.